src/main/java/com/stripe/param/AccountCreateParams.java

```diff
@@ -0,0 +1,2642 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class AccountCreateParams extends ApiRequestParams {
+  /**
+   * An [account token](https://stripe.com/docs/api#create_account_token), used to securely provide
+   * details to the account.
+   */
+  @SerializedName("account_token")
+  String accountToken;
+
+  /**
+   * Either a token, like the ones returned by [Stripe.js](https://stripe.com/docs/stripe.js), or a
+   * dictionary containing a user's bank account details.
+   */
+  @SerializedName("bank_account")
+  Object bankAccount;
+
+  /** Non-essential business information about the account. */
+  @SerializedName("business_profile")
+  BusinessProfile businessProfile;
+
+  /** The business type. Can be `individual` or `company`. */
+  @SerializedName("business_type")
+  String businessType;
+
+  /**
+   * Information about the company or business. This field is null unless `business_type` is set to
+   * `company`.
+   */
+  @SerializedName("company")
+  Company company;
+
+  /**
+   * The country in which the account holder resides, or in which the business is legally
+   * established. This should be an ISO 3166-1 alpha-2 country code. For example, if you are in the
+   * United States and the business for which you're creating an account is legally represented in
+   * Canada, you would use `CA` as the country for the account being created.
+   */
+  @SerializedName("country")
+  String country;
+
+  /**
+   * Three-letter ISO currency code representing the default currency for the account. This must be
+   * a currency that [Stripe supports in the account's country](https://stripe.com/docs/payouts).
+   */
+  @SerializedName("default_currency")
+  String defaultCurrency;
+
+  /**
+   * The email address of the account holder. For Standard accounts, Stripe will email your user
+   * with instructions on how to set up their account. For Custom accounts, this is only to make the
+   * account easier to identify to you: Stripe will never directly email your users.
+   */
+  @SerializedName("email")
+  String email;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
```

Comment:
Can you confirm whether we're always going to have this property on all params even if nothing is expandable?

Reply:
ahh thank you remi! no. we should not expect this. I will have to fix this in OpenAPI spec.

Reply:
It is likely easier to just have it always. Though I'm fine with not having it.

Reply:
Yup we currently have it as default except request with `DELETE` endpoint